### ISSUE TITLE
train loss

### ISSUE THREAD

**Issue by Wloner0809** [Open-Source-O1/Open-O1#3]
Thanks for your great work! I would like to ask why there are two sudden drops in this loss graph(copied from https://huggingface.co/O1-OPEN/OpenO1-LLama-8B-v0.1/blob/main/training_loss.png) Does this mean that the current stage of this work is overfitting on the O1-stype data format through sft?
![image](https://github.com/user-attachments/assets/a8adadf6-777a-4447-8c6a-7d40139d4817)

**Comment by gghfez** [comment]
That's likely to be 3 epochs and a high learning rate. 2nd and third time the model is exposed to the same data, it's familiar with it.
